data/outport/feeInfo.go

```diff
@@ -16,3 +16,13 @@ func (f *FeeInfo) SetGasUsed(gasUsed uint64) {
 func (f *FeeInfo) SetFee(fee *big.Int) {
 	f.Fee = fee
 }
+
+// SetGasRefunded sets the gas units refunded
+func (f *FeeInfo) SetGasRefunded(gasRefunded uint64) {
+	f.GasRefunded = gasRefunded
+}
+
+// SetHadRefund sets hadRefund field with true
+func (f *FeeInfo) SetHadRefund() {
```

Comment:
we usually have setters with value; is it better to pass value? there might be the need to set it to false